src/utils/resmoke/index.ts

```diff
@@ -50,15 +51,16 @@ const processResmokeLine = (line: string) => {
   const ports = getPorts(line) ?? "";
   const timeStamp = getTimeStamp(json) ?? "";
   const shellPrefix = getShellPrefix(json)?.padEnd(2, " ") ?? "";
+  const svc = getSvc(json) ?? "";
   const configSrv = getConfigServer(json)?.padEnd(8, " ") ?? "";
   const pid = getPid(json)?.padEnd(7, " ") ?? "";
   const ctx = getContext(json) ?? "";
   const msg = getMessage(json) ?? "";
   const attr = getAttributes(json) ?? "";
 
-  const output = `${resmokeFunction} ${ports}| ${timeStamp} ${shellPrefix} ${configSrv} ${pid} [${ctx}] "${msg}"${
-    attr ? `,${attr}` : ""
-  }`;
+  const output = `${resmokeFunction} ${ports}| ${timeStamp} ${shellPrefix} ${configSrv}${
+    svc ? ` [${svc}]` : ""
```

Comment:
Do you think the alignment metadata alignment can throw people off when some lines contain svc and some don't? This leads me to wonder, if there is a benefit to making lifted variables configurable?

Reply:
I think making the variables configurable would be a cool project idea! For this key, I just made it so that the non-svc logs look as they do now, in hopes of reducing any confusion

Reply:
Oh wait I think I misinterpreted your question! I'll ask about this in the UI channel on Monday I think